### PR TITLE
support ddl cases of encrypt scenario and bind some issue

### PR DESCRIPTION
**Pull request by JiekerTime**
bind issue #11103 #11105

### CODE REVIEW

File: shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/src/test/resources/cases/ddl/ddl-integration-test-cases.xml [hunk 1]
```diff
@@ -21,125 +21,128 @@
...
     </test-case>
-    
+
     <test-case sql="DROP INDEX order_index" db-types="PostgreSQL,Oracle" scenario-types="db,tbl,dbtbl_with_readwrite_splitting,readwrite_splitting">
         <assertion expected-data-file="unchanged_table.xml">
             <initial-sql sql="CREATE TABLE t_order_details(id int, description varchar(10));CREATE INDEX order_index ON t_order_details(description)" affected-table="t_order_details" />
         </assertion>
     </test-case>
-    
+
```
Comment (tristaZero) [inline shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/src/test/resources/cases/ddl/ddl-integration-test-cases.xml:60]: Hey, please revert these unnecessary formatting changes. BTW, this PR will solve the two issues you linked together, won't it?

**Review by tristaZero — CHANGES_REQUESTED** [review]

File: shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/src/test/resources/cases/ddl/ddl-integration-test-cases.xml (hunk 1 above)
Comment (JiekerTime) [reply shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/src/test/resources/cases/ddl/ddl-integration-test-cases.xml:60]: Not yet, the PR here just provides a way to review the issue.

Comment (tristaZero) [reply shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/src/test/resources/cases/ddl/ddl-integration-test-cases.xml]: To review these issues? So, these issues still need fixing or debugging?

**Review by tristaZero — APPROVED** [review]

**Comment by tristaZero** [comment]
So we still need to fix #11103 #11105? @JiekerTime You just added some test cases for helping to troubleshoot?
@strongduanmu @zjcnb

**Comment by zjcnb** [comment]
Ok, we are looking at it

**Comment by JiekerTime** [comment]
> So we still need to fix #11103 #11105? @JiekerTime You just added some test cases for helping to troubleshoot?
> @strongduanmu @zjcnb

This is true, besides I will help fix the issue if I can.

File: shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/src/test/resources/cases/ddl/ddl-integration-test-cases.xml (hunk 1 above)
Comment (JiekerTime) [reply shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/src/test/resources/cases/ddl/ddl-integration-test-cases.xml]: Yes, the issue is located in the rewritten part of the code of SS. Will be resolved in a later PR.
